test/unit/node/generated-code-dependencies.js

```diff
@@ -347,6 +346,13 @@ suite('context test', function() {
     });
   });
 
+  function checkFile(outDir, filename, expected) {
```

Comment:
Just assert.equal (I commentd about this before so maybe you answered this before?)